snmp-mixin/alerts/alerts.libsonnet

```
@@ -28,6 +28,18 @@
             },
             'for': '5m',
           },
+          {
+            alert: 'SNMPRestarted',
+            expr: 'sysUpTime < 60',
+            labels: {
+              severity: 'warning',
```

```suggestion
              severity: 'warning',
```
I would set it to info, as you don't know if restart is planned or not. Also, boot from power off state would also fire it, I would set it to info.

---

Makes sense